src/scss/grommet-core/_objects.layer.scss

```diff
@@ -111,6 +111,14 @@
       max-height: calc(100vh - #{double($inuit-base-spacing-unit)});
       max-width: calc(100vw - #{double($inuit-base-spacing-unit)});
       transform: translate(50%, 50%);
+
```

Comment:
Could be, I haven't dug much into the inner workings of grommet. In my testing in the codepen, I only saw the issue with center and not the other alignments. I think because the other alignments don't add any left/right spacing outside the layer. (Maybe that's also a bug?)